lib/Server.ts

```diff
@@ -23,8 +22,6 @@ export class Server {
     return new AppRunner().run(
       {
         loaderProperties: {
-          mainModulePath: join(__dirname, '..'),
```

Comment:
Are you sure about this one? I don't think it will default to this value otherwise.

Reply:
It will default to `@css:` which points at the CSS directory in the dependencies. The JSDoc also says that, unless adding new components that are only available elsewhere, the CSS directory is a good place to load from.